### DISCUSSION TITLE
Option to open YouTube links in PiP by default

### DISCUSSION THREAD

**Discussion by dmmagit** [libre-tube/LibreTube#3010] [Ideas]
I've added YouTube links to open by default with Libretube, which is great. I wish I could go into the Settings on Libretube and specify to open video links with PiP first. 

Thanks for considering!

**Comment by itsmmt305** [comment]
doesnt sound bad at all!

**Comment by Gabs-Unk** [comment]
sounds good
{    open with     }
    info
    background
   pip (float)

**Comment by luckkmaxx** [comment]
Im not used to open YT links, do you mean in a browser right?
Anyway I consider opening in PIP could be great, and you then can continue viewing the website while watching the related YT video.
